### PR TITLE
Lower the opacity of inactive terms

### LINKED ISSUE #15: Add active session styles
using the `.term_active` class in version 1 you can style the active session.
see [this](https://github.com/matheuss/hyper-simple-highlight-active-session) example

### PR DESCRIPTION
**Pull request by ynnoj**
Fixes #15.
screen shot 2017-01-11 at 15 47 25

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,5 +36,13 @@ exports.decorateConfig = config => Object.assign({}, config, {
...
 			border-color: rgba(255, 106, 193, 0.25);
 		}
+		.term_fit:not(.term_term) {
+		        opacity: 0.6;
+   		}
+    		.term_fit.term_active {
+      			opacity: 1;
+      			transition: opacity 0.12s ease-in-out;
+      			will-change: opacity;
+   		}
```
Comment (SamVerschueren) [inline index.js]: Incorrect indentation.

Comment (ynnoj) [inline index.js]: @SamVerschueren Which line? It follows the same indentation as the existing CSS rule 🤔

Comment (SamVerschueren) [inline index.js]: Look at the diff. You are using spaces instead of tabs. (line 40 - 46)

**Comment by SamVerschueren** [comment]
You might be still working on it (not sure). Just wanted to let you know it's still messed up.

**Comment by ynnoj** [comment]
@SamVerschueren I (think) we're there now. Atom was ignoring `.editorconfig` 😑

**Comment by sindresorhus** [comment]
Thank you :)

**Pull request merged**